src/components/common/ParentLoadingComponent.js

```diff
@@ -2,14 +2,14 @@ import { SmileFilled } from '@ant-design/icons';
 import { Layout, Spin } from 'antd';
 import PropTypes from 'prop-types';
 import React from 'react';
-import ParentNavSider from '../common/ParentNavSider';
+// import ParentNavSider from '../common/ParentNavSider';
 
```

Comment:
commented out parent nav sider to stop it from rendering when the parent dashboard loaded

Reply:
Nice catch!. I was wondering why it was rendering during loads. I am going to have to check this component out